app/controllers/provider_interface/offer_changes_controller.rb

```diff
@@ -58,16 +70,37 @@ def update
 
     def render_providers
       set_alternative_providers
+      @page_title = \
```

Comment:
The repeat conditionals in these `render_*` methods make me think `@page_title` generation could be wrapped up in a convenient method. I also wonder if the render methods are the right place to set this var.